backend/src/main/java/gov/cdc/usds/simplereport/db/model/auxiliary/SnomedConcept.java

```diff
@@ -0,0 +1,14 @@
+package gov.cdc.usds.simplereport.db.model.auxiliary;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class SnomedConcept {
```

Comment:
I checked into implementing this class with it... but because I need to access each attribute independently I ended up having to declare all the getters by hand. I would rather keep using the lombok annotations to save that effort unless there is another gain that I am missing to see. LMK your thoughts!

Reply:
So records provide [getters](https://www.baeldung.com/java-record-keyword#2-getters) by just doing `snomedConcept.propertyName()`.

This file would turn into just 
```java
package gov.cdc.usds.simplereport.db.model.auxiliary;

public record SnomedConcept(String name, String code, TestResult displayName) {
}
```

and then for getters it would just be 
```java
snomedConcept.code();
concept.displayName();
```